alf/algorithms/generator.py

```diff
@@ -104,12 +102,12 @@ def __init__(self,
             entropy_regularization (float): weight of entropy regularization
             kernel_sharpness (float): Used only for entropy_regularization > 0.
                 We calcualte the kernel in SVGD as:
-                    exp(-kernel_sharpness * reduce_mean((x-y)^2/width)),
+                    :math:'\exp(-kernel_sharpness * reduce_mean(\frac{(x-y)^2}{width}))'
```

Comment:
Make sure to change the format for every math equation and code chunk in this file

Reply:
Done for all modified files